packages/frontend/src/components/DataViz/store/cartesianChartBaseSlice.ts

```diff
@@ -116,6 +116,16 @@ export const cartesianChartConfigSlice = createSlice({
             config.display = config.display || {};
             config.display.yAxis = config.display.yAxis || [];
 
+            // NOTE: If there's only one y-axis, set the label on the series as well
```

Comment:
Could this be done in the CartesianChartModel so it resolves on render? Or is tied to state changes?

Reply:
That's a good point, let me try to do it on the `CartesianChartModel`!

Reply:
And done! much better